miasm/core/asmblock.py

```diff
@@ -117,6 +117,12 @@ def __str__(self):
     def addline(self, l):
         self.lines.append(l)
 
+    def offset(self):
+        """
+        returns the offset (as int) of the first line in the block
+        """
+        min([line.offset for line in block.lines])
```

Comment:
You don't need to create a list then compute the minimum value:
```suggestion
        min(line.offset for line in block.lines)
```